scenes/Game/Game.gd

```diff
@@ -0,0 +1,56 @@
+extends Control
+
+var root_scene_map = {
+	Constants.SceneId.StartMenu: preload("res://scenes/StartMenu/StartMenu.tscn"),
+	Constants.SceneId.World: preload("res://scenes/World/World.tscn"),
+	Constants.SceneId.PlanetSurface: preload("res://scenes/PlanetSurface/PlanetSurface.tscn"),
+}
+
+func _input(event) -> void:
+	if event.is_action_pressed("ui_cancel"):
+		$MainMenu.get_node("MainMenuPanel/ButtonContainer/NewGame").visible = (
+			GameState.scene != Constants.SceneId.StartMenu
+		)
+		
+		$MainMenu.visible = !$MainMenu.visible
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	GameState.connect("change_scene",self, "_change_scene")
+	get_tree().get_root().connect("size_changed", self, "_size_changed")
+	GameState.scene = Constants.SceneId.StartMenu
+	_size_changed()
+
+func _change_scene(sceneId):
+	print('_change_scene', root_scene_map, sceneId, root_scene_map.has(sceneId))
```

Comment:
Do you think we should remove the print statement before merging?

Reply:
Good catch. I'll fix.